### ISSUE TITLE
Ensime-Server crashes on unicode arrow

### ISSUE THREAD

**Issue by mpollmeier** [ensime/ensime-server#1069]
This is weired, but reproducible, at least for me: see [minimal sample repo](https://github.com/mpollmeier/ensime-crash-unicode)

As soon as you change the `=>` to the unicode arrow `⇒` in `CrashIt.scala`, ensime crashes with a parboiled error. To reproduce, just clone the above repo, regenerate the .ensime for your environment (e.g. run `sbt gen-ensime`) and start ensime. For me, it fails with the below error. My local .ensime and the full stack trace is also in the repo. 

```
13:52:22.316 DEBUG akka://ENSIME/user/ensime-main/tcp-server/con1/3 o.e.s.RequestHandler - received handled message StringResponse(package abc

object CrashIt extends App {
  Seq(1, 2).foreach { i ⇒
  }
}
) in state receive
13:52:23.035 ERROR akka://ENSIME/user/ensime-main/tcp-server/con1 o.e.s.t.TCPConnectionActor - Error seen during message processing, closing client connection
org.parboiled.errors.ParsingException: Invalid :
Unexpected end of input, expected Digits, Frac, Exp, Whitespace, Sexp or ) (line 7, pos 6):
")) 4
```

**Comment by edani** [comment]
Is this with the emacs client? This should be fixed by ensime/ensime-emacs#218

**Comment by mpollmeier** [comment]
yes that right! updating to the latest melpa fixed it. thank you!
